scipy/spatial/_spherical_voronoi.py

```diff
@@ -304,9 +304,8 @@ def _calculate_areas_2d(self):
         arcs = self.points[self._simplices] - self.center
 
         # Calculate the angle subtended by arcs
-        cosine = np.einsum('ij,ij->i', arcs[:, 0], arcs[:, 1])
-        sine = np.abs(np.linalg.det(arcs))
-        theta = np.arctan2(sine, cosine)
```

Comment:
Three angles calculated via `einsum` and a determinant seemed a bit much for calculating the radian angle on a circle.